README.md

```diff
@@ -64,6 +63,12 @@ This protocol is a master-slave protocol. The host has the role of _master_ and
 
 ```greentea-client``` implements the key-value protocol tokenizer and parser.
 
+## Greentea protocol sequence and programming guide
+
+The sequence of key-value events during a Greentea test and the use of the greentea-client API are explained in the documentation of [htrun](placeholder).
```

Comment:
Note: The placeholder should be replaced with a real link, after we finish refactoring docs for the greentea and htrun host tools.